go.mod

```diff
@@ -35,7 +36,6 @@ require (
 	github.com/invopop/jsonschema v0.12.0 // indirect
 	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/mailru/easyjson v0.7.7 // indirect
-	github.com/mr-tron/base58 v1.2.0 // indirect
```

Comment:
Kind of concerning that this was even indirect before, looking into the `mod why`

Reply:
Ah I see, looks like its from chain-selectors. Should be OK